README.md

```diff
@@ -60,6 +60,26 @@ See the [issue tracker] if you're unsure where to start, or feel free to reach o
 
 See also: the Kubernetes [community page].
 
+## Why kind?
+
+kind exists because we wanted something we were comfortable using for Kubernetes's own CI to test Kubernetes itself.
+
+ - kind doesn't use hyperkube or patch in binaries to existing images, we ~build a "real" kubernetes release, our images are more or less (soon to be fully) self-contained.
```

Comment:
this one is a rather specific callout, not sure if we should keep it as is :grimacing:

Reply:
What if we trimmed it down to "kind doesn't use hyperkube or patch in binaries to existing images"?

Reply:
thought: maybe the opposite direction? something along the lines of """"kind ships upstream release builds""" ? hyperkube is also in-and-of-itself fine (and pretty neat!), it's just less common I think, and we ship a very typical upstream build w/o any patching business.

Reply:
nit: s/ships/supports/ ?

Reply:
It does both, not sure which is clearer. 
We support building from apt, or building from k8s source in the same style as a release, out of the box though we also ship pre-built images that contain a release .. :man_shrugging: